frontend/src/components/Final/Final.js

```diff
@@ -46,10 +45,6 @@ const Final= ({ experiment, participant, session, score, final_text, action_text
         };
     }, [score, showScore]);
 
-    const finalize = () => {
-        finalizeSession({ session, participant });
-    }
-
```

Comment:
Was this removed by esLint? The `finalizeSession` logic certainly shouldn't go. It makes sure the backend receives the signal to write to the database that the session is finished, and its end time. (So we can easily find sessions in which participants completed the whole experiment.)

Reply:
Well, `eslint` warned me about it because it was an unused function and therefore an unused import. The `finalize` function never called anywhere. I don't know if that's always been the case?